perses/charts/values.yaml

```
@@ -0,0 +1,248 @@
+global:
+
+  ## Labels to apply to all resources
+  ##
+  commonLabels:
+perses:
+  # Default values for perses.
```

Enter an empty dict here to indicate the expected data type.
```suggestion
  commonLabels: {}

perses:
  # Default values for perses.
```